### PR TITLE
Fix CS in Changelogs in 5.1

### PR DESCRIPTION
**Pull request by jderusse**
| Q             | A
| ------------- | ---
| Branch?       | 5.1
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Component/Messenger/Bridge/Doctrine/Tests/Transport/DoctrineTransportTest.php [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <?php
 
 /*
-* This file is part of the symfony project.
-*
-* (c) Vincent Touzet <vincent.touzet@dotsafe.fr>
```
Comment (jderusse) [inline src/Symfony/Component/Messenger/Bridge/Doctrine/Tests/Transport/DoctrineTransportTest.php:6]: Spotted :rofl:

**Comment by nicolas-grekas** [comment]
Thank you @jderusse.

File: src/Symfony/Component/Messenger/Bridge/Doctrine/Tests/Transport/DoctrineTransportTest.php (hunk 1 above)
Comment (jderusse) [reply src/Symfony/Component/Messenger/Bridge/Doctrine/Tests/Transport/DoctrineTransportTest.php:6]: https://github.com/symfony/symfony/pull/30805 for reference

Comment (jderusse) [reply src/Symfony/Component/Messenger/Bridge/Doctrine/Tests/Transport/DoctrineTransportTest.php:6]: hmm.. this file were renamed and git history trapped me. It exists in 4.4 too :(

**Pull request merged**